GoogleForCreators/web-stories-wp
# Canvas to image improvements / Safari images fix

## Summary

This fixes Safari missing images issue due to unfinished image decoding (the library was waiting an arbitrary 100ms to take the "screenshot", after an exhaustive investigation it turns out that adding more delay (and doing various other things), only partially fixes the issue [it happened less often]).
Thanks to that discovery we can also remove the 100ms delay, speeding up all `html-to-image` usages (Presets/Text Sets smart color + Eyedropper + Thumbnails caching) 🎉 

## Relevant Technical Choices



## To-do

- [ ] Upstream PRs



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

Can be also tested on other browsers but the Safari is the main goal.
1. Enable thumbnails caching in the Experiments
2. Create a story and insert a few pages with images
3. Check if thumbnails are generated properly (images not missing)


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8923
